.gitignore

```diff
@@ -1,5 +1,7 @@
 .idea
 
+datastore/rough_pad.py
+
```

Comment:
Also using a proven `.gitignore`, such as the one from [`py-ipfs-http-client`](https://github.com/ipfs/py-ipfs-http-client/blob/master/.gitignore), is probably a good idea since it contains rules for a bunch of editors and build tools.